src/ogs.styl

```diff
@@ -256,6 +261,11 @@ dark.strong-loss                        = lighten(dark.loss, 25%)
 dark.weak-win                           = darken(dark.win, 25%)
 dark.weak-loss                          = darken(dark.loss, 25%)
 
+dark.strong-w-text                     = #3C3939
+dark.strong-l-text                     = #3C3939
+dark.weak-w-text                       = #E4DCDC
+dark.weak-l-text                       = #E4DCDC
```

Comment:
Text color in dart theme is #bbb. Would this work as well?

Reply:
updated to #bbb